common/src/main/java/com/wynntils/core/managers/ManagerRegistry.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.managers;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.commands.ClientCommandManager;
+import com.wynntils.core.config.ConfigManager;
+import com.wynntils.core.features.Feature;
+import com.wynntils.core.features.overlays.OverlayManager;
+import com.wynntils.core.keybinds.KeyManager;
+import com.wynntils.core.webapi.WebManager;
+import com.wynntils.mc.utils.CompassManager;
+import com.wynntils.mc.utils.CrashReportManager;
+import com.wynntils.wc.ModelManager;
+import java.lang.reflect.InvocationTargetException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.apache.commons.lang3.reflect.MethodUtils;
+
+public class ManagerRegistry {
+    private static final List<Class<? extends Manager>> PERSISTENT_MANAGERS = new ArrayList<>();
+    private static final Map<Class<? extends Manager>, List<Feature>> MANAGER_DEPENDENCIES = new HashMap<>();
+    private static final Set<Class<? extends Manager>> ENABLED_MANAGERS = new HashSet<>();
+
+    public static void init() {
+        registerPersistentDependency(CompassManager.class);
+        registerPersistentDependency(ConfigManager.class);
+        registerPersistentDependency(ClientCommandManager.class);
+        registerPersistentDependency(KeyManager.class);
```

Comment:
Do you think you can rename KeyManager into KeyBindManager while you're at it? I've lost count at how many times I need to check what kind of keys it's managing, thinking it's either dungeon keys, or SSH keys, but never realizing it's keybindings. :)

Reply:
Done